Develop

Wrestringer opened the pull request:
Созданы тесты для 4 сервисов:
* POST /api/v1/courier Создание курьера;
* POST /api/v1/courier/login Логин курьера;
* POST /api/v1/orders Создание заказа;
* GET /api/v1/orders Получение списка заказов.

Внимание! Для сервиса POST /api/v1/courier/login часть тестов провалена по причине некорректности работы сервиса https://qa-scooter.praktikum-services.ru/docs/#api-Courier-Login

FreeArtisan committed:
added README.md

FreeArtisan committed:
changed pom.xml

FreeArtisan committed:
added constants

FreeArtisan committed:
added utils to generate random string

FreeArtisan committed:
added models

FreeArtisan committed:
added steps

FreeArtisan committed:
added tests

FreeArtisan committed:
add allure report

nikolay-977 commented on src/test/java/ru/praktikum/services/qa/scooter/CourierCreateNegativeTest.java:
```diff
@@ -0,0 +1,60 @@
+import static ru.praktikum.services.qa.scooter.steps.StepsCourier.sendPostRequestToCreateCourier;
+import static ru.praktikum.services.qa.scooter.utils.Utils.randomString;
+
+public class CourierCreateNegativeTest {
```
Можно улучшить: применить наследование - создать класс BaseTest и вынести в него методы с аннотациями `@Before` и `@After`, где это возможно. Это позволит избежать повторяющегося кода и сократит код автотестов.

nikolay-977 commented on src/test/java/ru/praktikum/services/qa/scooter/constants/Colors.java:
```diff
@@ -0,0 +1,6 @@
+package ru.praktikum.services.qa.scooter.constants;
```
Необходимо исправить: В пакете src.**test**.java должны быть пакеты классов с кейсами. Остальные пакеты, такие как constants, models, steps, utils следует располагать в src.**main**.java.

FreeArtisan committed:
added BaseTest
constants, utils and models moved to src.main.java (steps cant be moved to main because its ude a ResAssured)
replaced the report with a new one

Wrestringer commented:
Исправлены указанные ошибки.
Добавлен класс BaseTest
Классы models, utils, constants перенесены в src.main.java (steps не может быть перенесен т.к. этот класс использует RestAssured).
Обновлен отчет Allure
